index.d.ts

```diff
@@ -106,628 +100,3416 @@ interface ClientOptions {
   }
 }
 
-declare class Client extends EventEmitter {
+declare class Client<RequestBody = any, Response = any, Context = unknown> extends EventEmitter {
```

Comment:
These generics on the class should not be needed since each method (rightfully) declares its own generics